frontend/index.html

```diff
@@ -180,38 +236,39 @@ <h5>Recommended stories for you!</h5>
         </li>
       </ul>
     </section>
-    <!-- Dropdown Stucture Template -->
-    <ul id="dropdown-nav" class="dropdown-content hoverable">
-      <li>
-        <a class="flow-text" href="#!" id="edit-interests">
-          <i class="material-icons">edit</i>Edit</a>
-      </li>
-      <li class="divider"></li>
-      <li>
-        <a class="tooltipped flow-text" data-position="left" data-delay="50" data-tooltip="Add Github account for more accurate news feed"
-          href="#!">
-          <i class="material-icons">person_pin</i>Profile</a>
-      </li>
-      <li>
-        <a class="flow-text" href="#!">
-          <i class="material-icons">settings</i>Settings</a>
-      </li>
-    </ul>
-    <!-- Modal Structure -->
-    <div id="modal-edit-interests" class="modal modal-fixed-footer">
-      <div class="modal-content">
-        <h4>Manage your interests</h4>
-        <p>You can add and delete your tags for personalizing newses to you. Type some texts for search our categories.</p>
-        <div class="chips chips-initial">
-        </div>
-      </div>
-      <div class="modal-footer">
-        <a href="#!" class="modal-action modal-close waves-effect waves-green btn-flat">Save</a>
+  <!--g-->
+  <!-- Dropdown Stucture Template -->
+  <ul id="dropdown-nav" class="dropdown-content hoverable">
+    <li>
+      <a class="flow-text" href="#!" id="edit-interests">
+        <i class="material-icons">edit</i>Edit</a>
+    </li>
+    <li class="divider"></li>
+    <li>
+      <a class="tooltipped flow-text" data-position="left" data-delay="50" data-tooltip="Add Github account for more accurate news feed"
+        href="#!">
```

Comment:
When the attributes are wrapped, there should be extra indentation for the wrapped line